static/gsApp/views/amCheckout/steps/productSelect.tsx

```diff
@@ -213,22 +226,67 @@ const ProductOptionContent = styled('div')<{gradientColor: string; enabled?: boo
     }
   }
 
-  &:hover {
-    background: linear-gradient(
-      0.33turn,
-      ${p => p.gradientColor},
-      ${p => p.theme.background},
-      ${p => p.gradientColor},
-      ${p => p.theme.background}
-    );
-    background-size: 400% 400%;
-    animation: gradient 4s ease-in-out infinite;
-    border-color: ${p => p.gradientColor};
-
-    button {
-      border-color: ${p => p.gradientColor};
-    }
+  img:nth-child(1) {
+    transition: transform 0.2s ease-out;
+
+    ${p =>
+      p.enabled &&
+      css`
+        transform: translate(-3px, 7px);
+      `}
   }
+
+  img:nth-child(2) {
+    transition: transform 0.2s ease-out;
+
+    ${p =>
+      p.enabled &&
+      css`
+        transform: translate(6px, -2px);
+      `}
+  }
+
+  img:nth-child(3) {
+    transition: transform 0.2s ease-out;
+
+    ${p =>
+      p.enabled &&
+      css`
+        transform: translate(0px, -5px);
+      `}
+  }
```

Comment:
you could write this cleaner using css variables, something like 
```suggestion
  --star-1-translate: 0, 0;
  --star-2-translate: 0, 0;
  --star-3-translate: 0, 0;
  
  ${p =>
    p.enabled &&
    css`
      --star-1-translate: -3px, 7px;
		  --star-2-translate: 6px, -2px;
		  --star-3-translate: 0px, -5px;`}

  img:nth-child(1) {
    transition: transform 0.2s ease-out;
    transform: translate(var(--star-1-translate));
  }

  img:nth-child(2) {
    transition: transform 0.2s ease-out;
    transform: translate(var(--star-2-translate));
  }

  img:nth-child(3) {
    transition: transform 0.2s ease-out;
    transform: translate(var(--star-3-translate));
  }
```